src/app/components/message/content/util.ts

```diff
@@ -4,7 +4,8 @@ import { decryptFile } from '../../../utils/matrix';
 export const getFileSrcUrl = async (
   httpUrl: string,
   mimeType: string,
-  encInfo?: EncryptedAttachmentInfo
+  encInfo?: EncryptedAttachmentInfo,
+  forceFetch?: boolean
```

Comment:
This function was made to decrypt file but maybe named wrong so I think we should use different function for fetching unencrypted file as adding a boolean param make it hard to reason what this function do.

Apart from it maybe in a different PR we can later remove the line: 15 createObjectURL outside it.

Reply:
actually I think I can fix it, I am currently working on fixing the download problem and has to changed getFileSrcUrl usage as I found we already have a `decryptFile` function in app/utils/matrix